### PR TITLE
Feature/tlvf add copy methods to uint8 lists

### PR DESCRIPTION
**Pull request by LiorAmram**
Add copy methods to uint8_t lists on tlvf, applying this feature: https://github.com/prplfoundation/prplMesh/issues/973

### CODE REVIEW

**Review by tomereli — APPROVED** [review]

**Review by morantr — APPROVED** [review]
Approved, but you need to remove the merge commits and rebase to the master instead.
Added "don't merge" label so you could fix it before auto-merge will take action.

**Pull request merged**
